src/config.c

```diff
@@ -384,6 +384,43 @@ static int updateOOMScoreAdjValues(sds *args, const char **err, int apply) {
     return C_OK;
 }
 
+/* Parse an array of sds strings, validate and populate server.client_obuf_limits
+ * if valid. Used in CONFIG SET and configuration file parsing.
+ * args: <class> <hard limit> <soft limit> <soft seconds> */
+static int updateClientOutputBufferLimit(sds *args, const char **err, int apply) {
+    int class = getClientTypeByName(args[0]);
+    unsigned long long hard, soft;
+    int hard_err, soft_err;
+    int soft_seconds;
+    char *soft_seconds_eptr;
+
+    if (class == -1 || class == CLIENT_TYPE_MASTER) {
+        if (err) *err = "Unrecognized client limit class: the user specified "
+                        "an invalid one, or 'master' which has no buffer limits.";
```

Comment:
```suggestion
        if (err) *err = "Invalid client class specified in buffer limit configuration.";
```

Reply:
Note that this error message will also be prompted in CONFIG SET